src/app.tsx

```diff
@@ -20,7 +21,9 @@ const App: FunctionComponent = () => {
         accounts.length > 0 ? setMetamaskConnected(true) : setMetamaskConnected(false);
       }
     } catch (e) {
-      e.code === 4001 ? setMetamaskConnected(true) : console.debug(e);
+      if (e instanceof EthereumProviderError) {
+        e.code === 4001 ? setMetamaskConnected(true) : console.debug(e);
+      }
```

Comment:
Good catch! Ill update it! 

Actually, Ill also change the other error handling to `log ? log(e.message) : console.error(e.message)` instead of https://github.com/OpenCerts/admin-website-v2/blob/a9146e73a8fa73c404d5fccb62192d2accf9aa17/src/components/util/issue.tsx#L22-L24 where it does nothing when the `log` object is undefined

Reply:
Yes, that would be good as well to default to `console.error(e.message)`

When `e` is not an instance of `Error`, maybe we can do this:
```typescript
log ? log("Unable to issue certificate", e) : console.error("Unable to issue certificate", e);
// or
log ? log("Unable to issue certificate", JSON.stringify(e)) : console.error("Unable to issue certificate", JSON.stringify(e));
```